Documentation/ceph-cluster-crd.md

```diff
@@ -1615,3 +1585,47 @@ data:
 
 Note: if you are using self-signed certificates (not known/approved by a proper CA) you must pass `VAULT_SKIP_VERIFY: true`.
 Communications will remain encrypted but the validity of the certificate will not be verified.
+
+### Deleting a CephCluster
+Deleting a CephCluster an uncommon-in-production but necessary part of a storage cluster's
+lifecycle. It is simple to initiate deletion of a CephCluster using `kubectl delete` just as with
+any Kubernetes resource.
+
+During deletion of a CephCluster resource, Rook protects against accidental or premature destruction
+of user data by blocking deletion if there are any other Rook-Ceph Custom Resources that reference
```

Comment:
Maybe it's personal preference, but qualifying things with "rook-ceph" usually seems redundant. 
```suggestion
of user data by blocking deletion if there are any other resources that reference
```

Reply:
IMO, Rook-Ceph can be a clarifying distinction compared Rook-Cassandra or Rook-NFS resources that a user might also have.

Reply:
All the docs are separated into a ceph section, but not a big deal.

Reply:
This is still a clearer distinction that Rook-Ceph will not block if there are references from other Rook backends (like Rook-Cassandra or Rook-NFS). Only references from Rook-Ceph will block. That is not necessarily implied by being in the Ceph section of the docs to those who haven't been part of the design discussions.

Reply:
I disagree that clarity is strictly needed since we're 1.5K lines into the ceph-cluster-crd doc and we're in the section "Deleting a CephCluster", but I won't block the PR on it either if it's your preference. :)

Reply:
It is needed, because this section is talking about things OTHER than the CephCluster CR. It is not talking about THIS CR (CephCluster), and when that is happening we have to be clear about the bounds and limitations of what OTHER things we are talking about so that users don't assume that their Rook-NFS CRs will block the CephCluster from being used if the Rook-NFS references it.